tutorials/rendering/renderers.rst

```diff
@@ -0,0 +1,315 @@
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Maximum                 | 8 per mesh. Can be       | 8 per mesh, 256 per view.| Unlimited.               |
+| SpotLights              | increased.               |                          |                          |
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Maximum                 | 8                        | 8                        | 8                        |
+| DirectionalLights       |                          |                          |                          |
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Maximum clustered       | N/A                      | N/A                      | 512. Can be increased.   |
+| elements                |                          |                          |                          |
++-------------------------+--------------------------+--------------------------+--------------------------+
+| PCSS for                | ❌ Not supported.        | ✔️ Supported.            | ✔️ Supported.            |
+| OmniLight and SpotLight |                          |                          |                          |
++-------------------------+--------------------------+--------------------------+--------------------------+
+| PCSS for                | ❌ Not supported.        | ❌ Not supported.        | ✔️ Supported.            |
+| DirectionalLight        |                          |                          |                          |
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Light projector         | ❌ Not supported.        | ✔️ Supported.            | ✔️ Supported.            |
+| textures                |                          |                          |                          |
++-------------------------+--------------------------+--------------------------+--------------------------+
+
+Global Illumination
+~~~~~~~~~~~~~~~~~~~
+
+See :ref:`doc_introduction_to_global_illumination` for more information.
+
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Feature                 | Compatibility            | Mobile                   | Forward+                 |
++=========================+==========================+==========================+==========================+
+| ReflectionProbe         | ⚠️ Planned.              | ✔️ Supported.            | ✔️ Supported.            |
+|                         |                          |                          |                          |
++-------------------------+--------------------------+--------------------------+--------------------------+
+| LightmapGI              | ⚠️ Planned.              | ✔️ Supported.            | ✔️ Supported.            |
```

Comment:
LightmapGI baking and rendering are both supported in Compatibility since 4.3.

Baking requires a GPU with RenderingDevice support (I'd mention that in a footnote).

Reply:
RenderingDevice is only mentioned briefly as an "other feature". It might be worth introducing the concept of RenderingDevice as an abstraction layer in the introduction. That means we can use "RenderingDevice support" in some places instead of the verbose "Vulkan, Direct3D 12, or Metal".